src/main/business/applicationFacade.js

```diff
@@ -310,6 +310,20 @@ class ApplicationFacade{
         return businessMQs.map(bMQ => this.#mqBusinessToFE(bMQ));
     }
 
+    /**
+     * return a list of meta question relevant to add to the new exam:
+     * - each question has at least 1 key and 4 distractors that are not used yet in the exam
+     * - the keys and distractors returned are not used yet in the exam
+     * @throws {Error} - If the user is not signed in or does not have the necessary permissions
+     * @return {MetaQuestion[]} all the relevant question in a meta question structure
+     */
+    async getMetaQuestionsForExam(data) {
```

Comment:
add a call to `validateParameters` so we know what is the received data
maybe @OfekN11 can do it for you

Reply:
This should be (and will be) done at the business level, at the implementation.

Thats due to the possibilty that more function will call it.

The validation should be done by the ones who use the data.